mkosi/resources/mkosi-initrd/mkosi.conf

```diff
@@ -53,18 +55,28 @@ KernelModulesInclude=
                     /dm-raid.ko
                     /dm-verity.ko
                     /dmi-sysfs.ko
+                    /drm_buddy.ko
                     /efi-pstore.ko
                     /efivarfs.ko
                     /erofs.ko
                     /ext4.ko
+                    /i2c-smbus.ko
+                    /intel_vsec.ko
```

Comment:
Probably doesn't hurt, but this doesn't seem generic for AMD and probably only makes sense for x86_64?

Reply:
On non x86-64 it simply won't be there and will be skipped. And yes this is intel specific but that just means we should also include the AMD modules, since this should be generic